contracts/upgradeable_contracts/erc20_to_erc20/ForeignBridgeErc677ToErc677.sol

```diff
@@ -40,6 +41,10 @@ contract ForeignBridgeErc677ToErc677 is ERC677Bridge, BasicForeignBridgeErcToErc
         return isInitialized();
     }
 
+    function claimTokensFromErc677(address _token, address _to) external onlyIfUpgradeabilityOwner {
```

Comment:
Sorry for misleading you in the #118 issue comment. Most probably the token will not be owned by the bridge contract so such changes will not work. Let's rollback introduction of this method int this contract.

Reply:
Reverted changes here https://github.com/poanetwork/poa-bridge-contracts/pull/219/commits/17c726688939eb6697d487c12c64394ae64d6668

Reply:
thanks. Sorry about this.